src/OrleansSQLUtils/Properties/AssemblyInfo.cs

```diff
@@ -21,4 +22,7 @@
 [assembly: InternalsVisibleTo("UnitTests")]
 [assembly: InternalsVisibleTo("TesterInternal")]
 [assembly: InternalsVisibleTo("UnitTestGrains")]
+[assembly: InternalsVisibleTo("Orleans.NonSiloTests")]
```

Comment:
Yeah I need to clean up NonSiloTests... did couple things wrong there. So questions to other 
[assembly: InternalsVisibleTo("Tester.SQLUtils")] , you added those because it is needed in order to compile or because AzureUtils PR did it? 
Because those two are not completely equal.